docs/toggleable-passes.md

```diff
@@ -10,6 +10,7 @@
 | `-Opre-reduce` `-Ono-pre-reduce` | Disabled | [pre-reduce](#pre-reduce) |
 | `-Osupercombinators` `-Ono-supercombinators` | Enabled  | [supercombinators](#supercombinators) |
 | `-Osimplify-main` `-Ono-simplify-main` | Disabled | [simplify-main](#simplify-main) |
+| `-Ocross-refs` `-Ono-cross-refs` | Disabled | [cross-refs](#cross-refs) | 
```

Comment:
I suggest `pre-reduce` and `pre-reduce-refs`.
Both are combined so that `-Opre-reduce -Opre-reduce-refs` is full compile-timme reduction of functions, `-Opre-reduce -Ono-pre-reduce-refs` is limited compile-time reduction and `-Ono-pre-reduce` is no compile-time reduction.

Reply:
Also it would be nice if `-Opre-reduce-refs` could imply `-Opre-reduce` if `-Ono-pre-reduce` was not already specified.
Maybe this would complicate too much the logic of which flags are enabled, both for the user and ion the codebase, but consider this option and choose whatever you think is best.